File: Praktikum 3. Kondisional/Soal 5/PRAK305-2310817310001-AvantioFierzaPatria.py
# Fungsi untuk mengkonversi detik menjadi jam, menit, dan detik
def konversi_detik(detik):
    jam = detik // 3600
    sisa_detik = detik % 3600
    menit = sisa_detik // 60
    detik = sisa_detik % 60
    return jam, menit, detik

# Membaca 5 input detik dari pengguna
inputs = []
for i in range(5):
    detik = int(input("Input: "))
    inputs.append(detik)

# Mengkonversi detik dan menampilkan hasilnya
for detik in inputs:
    jam, menit, detik = konversi_detik(detik)
    if jam >= 24:
        hari = jam // 24
        jam %= 24
        print(f"{hari} hari {jam:02}:{menit:02}:{detik:02}")
    else:
        print(f"{jam:02}:{menit:02}:{detik:02}")

File: Praktikum 3. Kondisional/Soal 5/PRAK305-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

// Fungsi untuk mengkonversi detik menjadi jam, menit, dan detik
void konversi_detik(int detik, int *jam, int *menit, int *detik_sisa)
{
    *jam = detik / 3600;
    int sisa_detik = detik % 3600;
    *menit = sisa_detik / 60;
    *detik_sisa = sisa_detik % 60;
}

int main()
{
    int inputs[5];

    // Membaca 5 input detik dari pengguna
    for (int i = 0; i < 5; i++)
    {
        printf("Input: ");
        scanf("%d", &inputs[i]);
    }

    // Mengkonversi detik dan menampilkan hasilnya
    for (int i = 0; i < 5; i++)
    {
        int jam, menit, detik;
        konversi_detik(inputs[i], &jam, &menit, &detik);

        if (jam >= 24)
        {
            int hari = jam / 24;
            jam %= 24;
            printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);
        }
        else
        {
            printf("%02d:%02d:%02d\n", jam, menit, detik);
        }
    }

    return 0;
}
